webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/ShowNodeDetailsFromNode.scala

```diff
@@ -80,7 +80,7 @@ object ShowNodeDetailsFromNode {
   sealed trait DisplayDetailsMode { def tab : Int}
   case object Summary    extends DisplayDetailsMode { override def tab = 0 }
   case object Compliance extends DisplayDetailsMode { override def tab = 1 }
-  case object System     extends DisplayDetailsMode { override def tab = 6 }
```

Comment:
I'm not sure if it was a mistake or on purpose

Reply:
The tab changed between first implem, and was originally in 7 position, but someone changed the tab order, but not here

Reply:
see #2997